### DISCUSSION TITLE
Error 404 for my site in GitHub pages

### DISCUSSION THREAD

**Discussion by VVasiladioti** [community/community#23661] [Other Feature Feedback, Questions, & Ideas]
<p>Hello, I am getting an error 404 in my site <a href="https://vvasiladioti.github.io/" rel="noopener nofollow ugc">https://vvasiladioti.github.io/</a></p>
<p>This is the repository (private)<br>
(<a href="https://github.com/VVasiladioti/VVasiladioti.github.io" rel="noopener nofollow ugc">https://github.com/VVasiladioti/VVasiladioti.github.io</a>)</p>
<p>Up until recently, it worked fine, I recently downgraded from a pro member, is that what caused the problem?</p>
<p>Any way to fix it?</p>
<p>Thanks in advance</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/vvasiladioti">@VVasiladioti</a>, welcome to the GitHub Support Community!</p>
<p>Downgrading from Pro is the issue, yes. Pages sites can only be served from private repositories on the Pro plan. You’ll either need to upgrade again or make the repository public to enable Pages for this repository.</p>
<p>Be aware that after doing either of these you may also need to enable Pages in the repository settings page before your site is available again.</p>

**Comment by VVasiladioti** [comment]
<p>Hi <a class="mention" href="/thomasshaped">@thomasshaped</a>,</p>
<p>Thank you for your quick reply. Problem solved :slight_smile:</p>
